### PR TITLE
added calculation of conference length based on received conferenceStartTime

### PR DESCRIPTION
**Pull request by nils-ohlmeier**

### CODE REVIEW

File: src/features/FeatureExtractor.js [hunk 1]
```diff
@@ -377,17 +378,21 @@ class FeatureExtractor {
...
             }
 
-            if (timestamp > this.conferenceEndTime) {
-                this.conferenceEndTime = timestamp;
+            if (timestamp > this.sessionEndTime) {
+                this.sessionEndTime = timestamp;
             }
         }
+
+        if (requestType === 'conferenceStartTimestamp' && requestObj) {
```
Comment (andrei-gavrilescu) [inline src/features/FeatureExtractor.js:393]: can this be moved to a separate handle function e.g. handleConferenceBla...?

File: src/database/FirehoseConnector.js [hunk 2]
```diff
@@ -187,6 +190,7 @@ class FirehoseConnector {
             meetingUniqueId,
             endpointId,
             sessionDuration,
+            conferenceDuration,
```
Comment (andrei-gavrilescu) [inline src/database/FirehoseConnector.js:193]: I know this isn't necessarily consistent with the rest of our fields, but would it make sense to keep the Ms at the end so it's more readable?

Comment (andrei-gavrilescu) [inline src/database/FirehoseConnector.js:193]: update the redshift-schema.sql file with the new field, also add it to our "Notes for next deploy" document

File: src/features/FeatureExtractor.js (hunk 1 above)
Comment (nils-ohlmeier) [reply src/features/FeatureExtractor.js:393]: I tried that, but then setting `this.conferenceStartTime` wasn't visible from outside the function call any more. That is why I reverted to this less ideal solution.

File: src/database/FirehoseConnector.js (hunk 2 above)
Comment (nils-ohlmeier) [reply src/database/FirehoseConnector.js:193]: Fair point. I'll make these changes.

File: src/features/FeatureExtractor.js (hunk 1 above)
Comment (nils-ohlmeier) [reply src/features/FeatureExtractor.js]: Ah. Very cool. Thanks that pointer helped me resolve this issue.

**Pull request merged**